boards/px4/fmu-v2/init/rc.board_extras

```diff
@@ -0,0 +1,35 @@
+#!nsh
+#
+# PX4 FMUv2 specific board init
+#
+#------------------------------------------------------------------------------
+#
+# UART mapping on FMUv2:
+#
+# UART1         /dev/ttyS0      IO debug
+# USART2        /dev/ttyS1      TELEM1 (flow control)
+# USART3        /dev/ttyS2      TELEM2 (flow control)
+# UART4
+# UART7                         CONSOLE
+# UART8         /dev/ttyS6      SERIAL4/TELEM4
+#
+#------------------------------------------------------------------------------
+#
+
+#
+# Bootloader upgrade
```

Comment:
Any reason not to do that in `defaults`?

Reply:
Nothing too compelling, just trying to keep some of the files dedicated to distinct functionality where possible. 
https://github.com/PX4/Firmware/pull/11044#discussion_r245705981

Reply:
I'd move it there then, because it will be faster and less likely to be interfered with by other already running modules.

Reply:
Bootloader upgrade moved to `boards/px4/fmu-v2/init/rc.board_defaults`.